packages/react/src/components/AccountSettings/ChangePassword/__tests__/ChangePassword.test.tsx

```diff
@@ -233,4 +254,60 @@ describe('ChangePassword', () => {
     expect(specialCharError).toBeDefined();
     expect(submitButton).toHaveAttribute('disabled');
   });
+
+  it('renders as expected with component overrides', async () => {
+    const { container } = render(<ChangePassword components={components} />);
```

Comment:
Can we check for the specific custom components?

Reply:
That's a good point. Do you mean testing for the custom labels that custom components use, or is there a better way?

Reply:
Yeah probably just testing for the custom label text

Reply:
Done in [1d6987f](https://github.com/aws-amplify/amplify-ui/pull/3075/commits/1d6987fbfd3a3642d17bb5c544e2453b68f75321)